api/src/index.ts

```diff
@@ -2,17 +2,24 @@ import "dotenv/config";
 import express, { Express } from "express";
 import { migrator } from "./database/config/databaseClient";
 import { redisClient } from "./database/config/redisClient";
-import cors from 'cors';
+import cors from "cors";
+import helment from "helmet";
```

Comment:
Typo in `helment`.

Reply:
Thanks. Commit: 5407ff0a58f6e756de0c895c64d6b62931bcea9a